action.yaml

```diff
@@ -12,7 +12,7 @@ runs:
       with:
         separator: ","
     - name: Power BI VC Utils
-      uses: nathangiusti/Power-BI-VC-Utils@v2.3
+      uses: nathangiusti/Power-BI-VC-Utils@v2.4
       with:
```

Comment:
This new VC utils takes in a separator argument right? Do we need to add that argument here?

Reply:
You're right, the 2.4 version takes that in

Reply:
So you need to do the opposite. 

You need to add the argument back into the tj actions

And then also pass the separator argument to our Power BI VC Utils action. 

The beauty of the composite action is that it doesn't matter what the separator is for the two actions, they just need to match. So instead of forcing the user to put those arguments in, we do it for them and we have arbitrarily chosen commas because commas generally don't appear in file paths. If someone committed a file path with a comma that would actually break stuff but nothing is perfect.